client/src/components/IntroScreen.js

```diff
@@ -10,22 +10,23 @@ export function IntroScreen() {
             className="introScreenContainer"
             animate={controls}
         >
-            <motion.div 
+            <motion.div
                 className="introScreenContent"
                 animate={contentControls}
             >
-                <h1>This is a visualization of all the protests against Citizenship Amendment Act (CAA) across India</h1>
-                <motion.button 
+                <h1>India has been on the streets this winter to save the secular fabric of the country</h1>
+                <h2>Over 20 civilians have died since these protests began on 20th of December 2019. Here is a map of the protests across the country.</h2>
+                <motion.button
```

Comment:
This much text is giving a scroll on mobile and pushing the CTA below scroll, can we reduce this?

Reply:
Actually scratch that